Translation corrections (#114)
07th-mod/higurashi-console-arcs: Update/some_009.txt

	ClearMessage();

//rなんて、…なんて簡単で、……とても気楽な解決手段…！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "なんて、…なんて簡単で、……とても気楽な解決手段…！",
			NULL, "How... how easy...... What a simple solution...!", Line_Normal);
	ClearMessage();

//r罪悪感？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "罪悪感？",
			NULL, "Guilt?", Line_Normal);
	ClearMessage();

//r良心の呵責？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "良心の呵責？",
			NULL, "Pricks of conscience?", Line_Normal);
	ClearMessage();

//rあはははははははは、そんなものが何になるの？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "あはははははははは、そんなものが何になるの？",
			NULL, "Ahahahahahahahaha, what good would any of that do?", Line_Normal);
	ClearMessage();

//r畠山のおじーちゃん、…いや、あんな祟りに怯えて錯乱した、おかしなじーさんのことなんてかばったりしなければ、山瀬さんは生命を落さずに済んだんじゃないか。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "畠山のおじーちゃん、…いや、あんな祟りに怯えて錯乱した、おかしなじーさんのことなんてかばったりしなければ、山瀬さんは生命を落さずに済んだんじゃないか。",

> 			NULL, "If I hadn't stood up for old man Hatakeyama... no, that crazy old man who went insane from fear of the curse, Yamase-san wouldn't have had to die, would she?", Line_Normal);

Reviewer: ```suggestion
			NULL, "If I hadn't stood up for old man Hatakeyama... that crazy old man who went insane from fear of the curse, Yamase-san wouldn't have had to die, would she?", Line_Normal);
```

//rいや、…違う。私は『喋った』んじゃない。『捨てた』んだ、雛見沢のことを。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "いや、…違う。私は『喋った』んじゃない。『捨てた』んだ、雛見沢のことを。",

> 			NULL, "No... that's not right. I'm didn't <i>tell her</i> about Hinamzawa. I <i>threw away</i> Hinamizawa.", Line_Normal);

Reviewer: ```suggestion
			NULL, "No... that's not right. I didn't <i>tell her</i> about Hinamzawa. I <i>threw away</i> Hinamizawa.", Line_Normal);
```

//r大切な友達。大好きなひと。私を幸せにしてくれる、ありとあらゆるモノとヒト。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "大切な友達。大好きなひと。私を幸せにしてくれる、ありとあらゆるモノとヒト。",
			NULL, "My dear friends. The person I love. Everything and everyone that makes me happy.", Line_Normal);
	ClearMessage();

//rそのためには、これまでよりも心を硬く、強くして…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そのためには、これまでよりも心を硬く、強くして…。",
			NULL, "For that reason, I'll harden my heart and strengthen it more than ever...", Line_Normal);
	ClearMessage();

//r雛見沢なんて、…捨てて、しまえ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "雛見沢なんて、…捨てて、しまえ。",
			NULL, "Hinamizawa... was no longer my problem.", Line_Normal);
	ClearMessage();

//rそして、ひぐらしのなく声が少しずつ小さくなり、…朱の光が闇の中へと溶け込むこの瞬間に、私は。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして、ひぐらしのなく声が少しずつ小さくなり、…朱の光が闇の中へと溶け込むこの瞬間に、私は。",

> 			NULL, "And at the moment the sound of the higurashi gradually faded away... where I sat amidst the vermillion light fading into darkness...", Line_Normal);

Reviewer: ```suggestion
			NULL, "And in this moment where the sound of the higurashi gradually faded away... where I sat amidst the vermillion light fading into darkness...", Line_Normal);
```
